src/transformStateWithClones.js

```diff
@@ -7,7 +7,34 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const cloneState = JSON.parse(JSON.stringify(state));
+  const clonArr = [];
+
+  for (const action of actions) {
+    switch (action.type) {
+      case ('addProperties'):
+        Object.assign(cloneState, action.extraData);
+        break;
+
+      case ('clear'):
+        for (const key in cloneState) {
+          delete cloneState[key];
+        }
+        break;
+
+      case ('removeProperties'):
```

Comment:
```suggestion
      case 'addProperties':
        Object.assign(cloneState, action.extraData);
        break;

      case 'clear':
        for (const key in cloneState) {
          delete cloneState[key];
        }
        break;

      case 'removeProperties':
```
I think brackets are unnecessary